app/assets/javascripts/components/authentication/login_form.es6.jsx

```diff
@@ -44,6 +51,14 @@ class LoginForm extends Component {
     }.bind(this);
   }
 
+  renderErrorMessage() {
```

Comment:
I think we can just called this `renderMessage` since there won't be any others on the `LoginForm`!